src/components/Chat/ChatMessage.tsx

```diff
@@ -13,6 +26,9 @@ class ChatMessage extends UIComponent<any, any> {
   static propTypes = {
     as: customPropTypes.as,
 
+    /** Avatar */
+    avatar: PropTypes.object,
```

Comment:
it should be `customPropTypes.itemShorthand`

Reply:
done